tests/escaping/src/test/kotlin/test/EscapingTest.kt

```diff
@@ -0,0 +1,23 @@
+package test
+
+import com.apollographql.apollo3.ApolloClient
+import com.apollographql.apollo3.api.Optional
+import reserved.WhileQuery
+import reserved.type.Do
+
+class EscapingTest {
+  // Not actually a test, we're just making sure this compiles
+  suspend fun test() {
+    val response = ApolloClient.Builder().build()
+        .query(WhileQuery(
+            `return` = "public",
+            `for` = Do(
+                `if` = Optional.Absent,
+                yield_ = Optional.Absent,
```

Comment:
Could this be 
```
`yield` = Optional.Absent,
```
 ? instead?

Reply:
Ideally yes, but don't know how to do it, since the escaping on this one is not done by KotlinPoet but by us.

Reply:
Feels like that deserves another KotlinPoet PR? 'Do not escape things that are already escaped'

That won't solve it in the short term but maybe that can give us a better idea of what's going on?

Reply:
Good idea, I'll have a look!

Reply:
Turns out, this double escaping handling was already... handled 😅, but in the latest version, so I bumped it. Now we can have `` `yield` `` 🎉  (3323c10bda99713d8812ca39e244777f14179a18)